cirq/google/engine/engine.py

```diff
@@ -388,6 +399,32 @@ def _infer_gcs_results(self, job_config: JobConfig) -> None:
             job_config.gcs_results = '{}jobs/{}'.format(job_config.gcs_prefix,
                                                         job_config.job_id)
 
+    def _make_request(self, request: HttpRequest) -> Dict:
+        RETRYABLE_ERROR_CODES = [500, 503]
```

Comment:
nit: since this isn't a global constant this should just be lower case snake case